### ISSUE TITLE
import/order incorrectly classifies linked in modules.

### ISSUE THREAD

**Issue by jsg2021** [benmosher/eslint-plugin-import#882]
I have a project where I link in a library I'm working on at the same time as the app... the import/oder lint rules change based on linked state... even though the library's link is under node_modules.

**Comment by danny-andrews** [comment]
.eslintrc:
```yaml
rules:
  import/order:
    - error
    - groups: [builtin, external, internal, [sibling, index]]
      newlines-between: always
```

package.json:
```json
{
  "dependencies": {
    "@own-local/platform-dtd": "link:../platform-dtd"
  }
}
```

Code:

```js
import { injectIntl } from 'react-intl';
import type { Category, Business as BusinessType } from '@own-local/platform-dtd/lib/types';
```

Expected: no error

Actual: `There should be at least one empty line between import groups (import/order)`

**Comment by danny-andrews** [comment]
Update: Looks like this is caused by the hyphen in our scoped module name: `@own-local`. This PR fixes the issue! https://github.com/benmosher/eslint-plugin-import/pull/744

**Comment by jsg2021** [comment]
I don't have scoped modules.

**Comment by danny-andrews** [comment]
Ah, bummer, was hoping you had the same problem. Best of luck, mate!

**Comment by jsg2021** [comment]
Closing because it's irrelevant now
